### ISSUE TITLE
Packetbeat lowercases Elasticsearch ingest pipeline names 

### ISSUE THREAD

**Issue by Winterflower** [elastic/beats#26570]
Packetbeat version: `packetbeat-7.13.2-amd64.deb`

Hi team, 
we are using an Ingest Pipeline for our DGA project.
The name of the pipeline is `ML_DGA_pipeline`. 
We add the name of this pipeline in the config stanza in `packetbeat.yml` like this

```
output.elasticsearch:
  hosts: ["your-hostname:your-port"]
  pipeline: pipeline ML_DGA_ingest_pipeline
```

When we try to run packetbeat to ingest documents through this pipeline, packetbeat logs report an error `pipeline ml_dga_ingest_pipeline does not exist` - note that the pipeline name has been lowercased.
Note that Elasticsearch itself treats `ML_DGA_ingest_pipeline` and `ml_dga_ingest_pipeline` as two separate entities.

**Comment by legoguy1000** [comment]
This may be a libbeat issue as a whole, https://github.com/elastic/beats/blob/8ca57d309f1cabe95c08a29dd5a4c682a999fec5/libbeat/outputs/elasticsearch/client.go#L354

**Comment by kaiyan-sheng** [comment]
Or maybe we can add a note in the documentation first?

**Comment by legoguy1000** [comment]
I would say it should match or may cause further confusion. Looking at the logstash plugin, it doesn't seem to be doing any lowercasing, https://github.com/logstash-plugins/logstash-output-elasticsearch/blob/476097712b67e0452099769b2c8b0ef6e63e40eb/lib/logstash/outputs/elasticsearch.rb#L421.

**Comment by P1llus** [comment]
I think simply changing it would break many peoples current implementation that might already expect this to be lowercase only though

**Comment by kaiyan-sheng** [comment]
@P1llus Hmm if the user is only using lowercase anyway, then the change won't break them right?

**Comment by P1llus** [comment]
Thats correct, but it might be the other way around. If people know its always lowercase, thrn maybe someone used mixed or uppercase on the config side.

Maybe it's just me being picky though @kaiyan-sheng

**Comment by legoguy1000** [comment]
This has popped up in Fleet now, https://discuss.elastic.co/t/ingest-pipeline-id-not-working-for-udp-integration/296668?u=legoguy1000

**Comment by andrewkroh** [comment]
If the lowercasing functionality is removed then like @P1llus said the users that will be broken are ones that have incorrectly specified their pipeline name in the config. I would prefer to have those config be broken by the next upgrade than live with what I consider a bug.

It would be interesting to look back a git history and see if there was a reason the lowercasing was added.
